indexer/feature.cpp

```diff
@@ -375,6 +376,15 @@ void FeatureType::ParseMetadata() const
   m_metadataParsed = true;
 }
 
+void FeatureType::ParseAltitude() const
+{
+  if (m_altitudeParsed)
+    return;
+
+  m_pLoader->ParseAltitude();
```

Comment:
Do you have any idea where a good place for keeping an instance of AltitudeLoader is? 

For the time being I'm thinking about adding AltitudeLoader * m_altitudeLoader; / SharedLoadInfo::CreateAltitudeLoader() / SharedLoadInfo::GetAltitudeLoader()